src/python/turicreate/toolkits/_tf_utils.py

```diff
@@ -14,8 +14,9 @@ def get_gpu_names():
 	"""
 	Gets the available GPU names.
 	"""
-	import tensorflow as tf
-	gpu_names = tf.config.experimental.list_physical_devices('GPU')
+	import tensorflow.compat.v1 as _tf
+	_tf.disable_v2_behavior()
```

Comment:
If we're writing new code, we should be using the new API.

Does the following work?
```
import TensorFlow as tf
tf.config.experimental.list_physical_devices
```

Reply:
Yeah the new one works. Yeah that does make sense which is why I had that earlier. I will revert back this change.